client/app/(tabs)/newStoryScreen.tsx

```diff
@@ -5,18 +5,10 @@ import { View } from 'react-native';
 
 export default function newStoryScreen() {
   return (
-    <View style={styles.container}>
+    <View className='flex-col justify-around flex-1 items-center py-10'
```

Comment:
ive also used a margin of m-8 for the screens if we want to keep it consitant

Reply:
Actually - depending on how your background coloring works, it wont cover the whole page if you give it a marging - so can leave for now